lib/defines.h

```diff
@@ -179,20 +179,11 @@ static inline void memzero(void *ptr, size_t size)
 # define SEEK_END 2
 #endif
 
-#if HAVE_TERMIOS_H
-# include <termios.h>
-# define STTY(fd, termio) tcsetattr(fd, TCSANOW, termio)
-# define GTTY(fd, termio) tcgetattr(fd, termio)
-# define TERMIO struct termios
-# define USE_TERMIOS
-#else				/* assumed HAVE_TERMIO_H */
-# include <sys/ioctl.h>
-# include <termio.h>
-# define STTY(fd, termio) ioctl(fd, TCSETA, termio)
-# define GTTY(fd, termio) ioctl(fd, TCGETA, termio)
-# define TEMRIO struct termio
-# define USE_TERMIO
```

Comment:
As you remove `USE_TERMIO` definition you can also remove its usage (i.e. `sulogin.c`).

Reply:
Thanks!  Also removed USE_TERMIOS, since now it's always true.

Reply:
You forgot to remove `# define USE_TERMIOS` from `defines.h`